config/locales/application_form.yml

```diff
@@ -803,6 +803,33 @@ en:
               in_future: Enter a year before %{date}
             other_grade:
               blank: Enter your grade
+        candidate_interface/science_gcse_grade_form:
+          attributes:
+            single_award_grade:
+              blank: Enter your grade
+              invalid: Enter a real grade
```

Comment:
Has this copy from the designs? Should this be "Enter a valid grade"?

Reply:
This is the error message used in the current / old flow.  @paulrobertlloyd can you confirm?

Reply:
Hmmm, ‘real’ does seem odd, but does seem more like what somebody would say in real life. I’d stick with real if that’s what we’re using elsewhere.